nbd/src/util.cpp

```diff
@@ -278,6 +280,37 @@ int get_mapped_info(int pid, NBDConfig *cfg) {
     return 0;
 }
 
+int check_mount_status(const NBDConfig *cfg) {
+    std::ifstream ifs;
+    ifs.open("/proc/mounts", std::ifstream::in);
+
+    if (!ifs.is_open()) {
+        dout << "curve-nbd: failed to open /proc/mounts" << std::endl;
+        return -EINVAL;
+    }
+
+    std::string device, mountPath;
+    bool mounted = false;
+    while (ifs >> device >> mountPath) {
```

Comment:
Each line of `/proc/mounts` has six columns, so the input in `while condition` has a problem.
In the first time, `device` stores the first column, and `mountPath` stores the second column, but the next time, `device` will store the third column - filesystem type, and `mountPath` will store the fourth column which is the mount options.

Reply:
> Each line of `/proc/mounts` has six columns, so the input in `while condition` has a problem.
> In the first time, `device` stores the first column, and `mountPath` stores the second column, but the next time, `device` will store the third column - filesystem type, and `mountPath` will store the fourth column which is the mount options.

yeap, it's a **BUG**, i will fix it.